main.js

```diff
@@ -59,41 +59,24 @@ function createWindow() {
             global.config = c;
             // and load the index.html of the app.
             //Also setup the mod manager.
-            try {
-                mod_manager.Setup();
-                mainWindow.loadFile(path.resolve(__dirname, "index.html"));
-            }
-            catch(e) {
-                dialog.showMessageBox({
-                    type: "error",
-                    title: "Startup Error - Main Window Load",
-                    message: e.toString() + majorErrorMessageEnd,
-                    buttons: ["OK"]
-                }).then((button) => {
-                    app.quit();
-                });
-            }
+            return mod_manager.Setup().then(() => {
```

Comment:
Because migration happens on mod manager setup, this holds the window from loading its page. Please move the calling of this to after the window page is loaded.